bin/run-ci

```diff
@@ -1,6 +1,26 @@
 #!/bin/bash
 set -e
 
+function get_debugging_info {
+  echo "Failed to deploy Airflow!"
+  echo "Printing description and logs where containers in pod are not ready..."
+  for pod in $(kubectl get pods -n airflow | grep -v NAME | grep -v 1/1 | grep -v 2/2 | grep -v Completed | awk '{ print $1 }'); do
+    echo "======================="
+    set -x
+    kubectl describe pod -n airflow $pod
+    kubectl logs -n airflow $pod | tail -n 30
+    set +x
+    echo "======================="
+  done
+  kubectl get pods --all-namespaces
+  echo "======================="
+  kubectl get ds --all-namespaces
+  echo "======================="
+  kubectl get sts --all-namespaces
+  echo "======================="
+  kubectl get deployments --all-namespaces
+}
+
```

Comment:
I added this block to help with debugging CI failures. this is redundant with bin/run-ci in the Astronomer chart and they will be de-duplicated soon enough.